src/rspecTests.ts

```diff
@@ -192,7 +189,8 @@ export class RspecTests extends Tests {
     this.log.info(`Running test file: ${testFile}`);
     const spawnArgs: childProcess.SpawnOptions = {
       cwd: this.workspace.uri.fsPath,
-      shell: true
+      shell: true,
+      env: this.getProcessEnv()
```

Comment:
This fixes a bug that debug fails (due to lack of `EXT_DIR` environment variable).